app/src/main/java/science/apolline/view/activity/MainActivity.kt

```diff
@@ -509,7 +531,8 @@ class MainActivity : RootActivity(), NavigationView.OnNavigationItemSelectedList
         internal lateinit var mPrefs: SharedPreferences
         internal val EXTRAS_DEVICE_NAME = "DEVICE_NAME"
         internal val EXTRAS_DEVICE_ADDRESS = "DEVICE_ADDRESS"
-
+        internal val DISCONNECTED_SENSOR_DISPLAY_TITLE = "Capteur déconnecté"
```

Comment:
No, use the Internationalisation framework instead of constants

Reply:
I don't see what you mean, can I have an example please ?

Reply:
https://developer.android.com/guide/topics/resources/localization

Reply:
k thanks